patches/kernel-5.11.patch

```diff
@@ -334,10 +334,7 @@ diff --git a/kernel-dkms/nvidia/nvidia.Kbuild b/kernel-dkms/nvidia/nvidia.Kbuild
 index 90579de..c6f3614 100644
 --- a/kernel-dkms/nvidia/nvidia.Kbuild
 +++ b/kernel-dkms/nvidia/nvidia.Kbuild
-@@ -172,6 +172,7 @@ NV_CONFTEST_SYMBOL_COMPILE_TESTS += is_export_symbol_present_tegra_dce_client_ip
- NV_CONFTEST_SYMBOL_COMPILE_TESTS += is_export_symbol_present_dram_clk_to_mc_clk
- NV_CONFTEST_SYMBOL_COMPILE_TESTS += is_export_symbol_present_get_dram_num_channels
- NV_CONFTEST_SYMBOL_COMPILE_TESTS += is_export_symbol_present_tegra_dram_types
+@@ -172,3 +172,4 @@
 +NV_CONFTEST_SYMBOL_COMPILE_TESTS += is_export_symbol_present_close_fd
```

Comment:
Do this work at all? With so little context I'm kinda doubtful

Reply:
I changed it so it would apply on 455 as well since there are no such symbols on that branch and i tested that it indeed does apply and builds / loads fine too.

Reply:
Ok cool.